bitnami/fluentd/values.yaml

```diff
@@ -107,6 +107,9 @@ forwarder:
   ## @param forwarder.daemonGroup Fluentd forwarder daemon system group
   ##
   daemonGroup: root
+  ## @param forwarder.automountServiceAccountToken Mount Service Account token in pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
Is it ok to be true by default in this case?

Reply:
Yes, because it has RBAC rules